WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -634,6 +640,10 @@ private extension NotificationsViewController
             DDLogSwift.logError("Error refreshing Notification Row \(error)")
         }
     }
+
+    func setNeedsReloadResults() {
```

Comment:
Yet the variable could still be accessed directly throughout the class and other extensions being defined on the class. The setter seems unnecessary and not too swifty to me.  Not a huge deal if you prefer to keep it.